packages/autorest.python/autorest/codegen/templates/setup.py.jinja2

```diff
@@ -91,6 +91,11 @@ setup(
         {% else %}
         "{{ dependency_azure_core }}",
         {% endif %}
+        {% if extra_depends is defined %}
```

Comment:
nit: I think `extra_dependencies` is better

Reply:
yes, changed to "extra_dependencies"